test_app/apps.py

```diff
@@ -1,6 +1,14 @@
+import signal
+
 from django.apps import AppConfig
 
+from ansible_base.lib.middleware.logging import LogTracebackMiddleware
+
 
 class TestAppConfig(AppConfig):
     default_auto_field = 'django.db.models.BigAutoField'
     name = 'test_app'
+
+    def ready(self):
+        super().ready()
+        signal.signal(signal.SIGSYS, LogTracebackMiddleware.handle_signal)
```

Comment:
Based on the docs:

https://docs.djangoproject.com/en/5.1/topics/http/middleware/

It seems like connecting the signals would be better done in `LogTracebackMiddleware.__init__`, as opposed to requiring the extra step.

Reply:
1) not 100% following how to do that yet
2) Exactly what signal the app will receive on graceful timeout will differ....so how would this be handled